app.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express')
+const { getTopics } = require('./controllers/controller')
+
+const app = express()
+
+app.get('/api/topics', getTopics)
+
+
```

Comment:
You could include a server error handler in your app file (you wouldn't need to test this)